src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();
```

Comment:
it's a working solution but consider have Map where the key is the fruit name and the value is the fruit quantity. updating such storage would be easier